libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/SimpleCamera.kt

```diff
@@ -11,20 +11,20 @@ import com.mapbox.turf.TurfConstants
 import com.mapbox.turf.TurfMisc
 
 /**
- * The default camera used by [MapboxNavigation].
+ * The base camera implementation.
  */
 open class SimpleCamera : Camera() {
 
     companion object {
         /**
          * Default tilt angle for the Camera
          */
-        protected const val DEFAULT_TILT = 50
```

Comment:
Isn't this breaking SemVer?

Reply:
Same here and above - we're making a behavioral change that does impact the default experience, but we believe that the previous behavior was buggy and this is a clear improvement.

Reply:
We should be very transparent in the release notes though.